lib/jmap/calendar/blob_found_not_parsable.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'blob_found_not_parsable.g.dart';
+
+@JsonSerializable()
+class BlobFoundNotParsable with EquatableMixin {
+  final String? accountId;
```

Comment:
It is recommended that `AccountId` replace the data type `String` to make the object more meaningful and easier to use later.